src/main/antlr4/org/reaktivity/nukleus/maven/plugin/internal/parser/Nukleus.g4

```diff
@@ -287,6 +287,7 @@ list_member
    | KW_REQUIRED? integer_array_member SEMICOLON
    | KW_REQUIRED? varint_array_member SEMICOLON
    | KW_REQUIRED? array_member SEMICOLON
+   | variant_array SEMICOLON
```

Comment:
Can `variant_array` be `required` too?

If not needed for immediate tasks, let's file an enhancement to track the missing capability for now.

Reply:
You are right, one of the list fields in `protocol.idl` has `required` keyword for `variant_array`, so I added `KW_REQUIRED` on the grammar.